includes/migration-guide/runtime/xml/xslt-style-sheet-exception-message-changed.md

```diff
@@ -12,8 +12,15 @@ Update any app code depending on the exception message from this error condition
 |:--------|:------------|
 | Scope   |Edge|
 |Version|4.5|
-|Type|Runtime
+|Type|Runtime|
 
 #### Affected APIs
```

Comment:
@gewarren, There doesn't seem to be a comment parallel to this heading. I *think* this comment is used by some tooling to detect compatibility between versions. Can you confirm if it's required or not?

Reply:
Yes, I believe it's used by an analyzer to detect compatibility issues. It is required.

Reply:
@gewarren Is there some tooling which can automatically add the comment wherever the Affected APIs heading exist?

Reply:
Not that I'm aware of.

Reply:
@gewarren, I'll try to make a tool. Not sure if I'll get to a good result or not.